### PR TITLE
DES-13573 - upgrade to CEFv64

### PR DESCRIPTION
**Pull request by rbarros-daitan**
CEF upgrade to CEFv64. 
It is needed to upgrade to VS2015 due to CEFGlue.
Some parameters from PopupFeatures got removed so changes had to be reflected on paragon code:
-minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindowManager.cs

### CODE REVIEW

File: minuet/CefGlue/Interop/Structs/cef_popup_features_t.cs [hunk 1]
```diff
@@ -21,12 +21,6 @@ internal unsafe struct cef_popup_features_t
         public int menuBarVisible;
         public int statusBarVisible;
         public int toolBarVisible;
-        public int locationBarVisible;
```
Comment (rbarros-daitan) [inline minuet/CefGlue/Interop/Structs/cef_popup_features_t.cs:24]: Parameters removed from popoutfeatures. @lneir

File: minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindowManager.cs [hunk 2]
```diff
@@ -121,12 +121,6 @@ public void BeforeApplicationWindowPopup(BeforePopupEventArgs eventArgs)
                 };
 
                 options.InitialState = "normal";
-                if (eventArgs.PopupFeatures.Fullscreen)
```
Comment (rbarros-daitan) [inline minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindowManager.cs:124]: PopupFeatures has some parameters removed. @lneir

File: minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindowManager.cs [hunk 3]
```diff
@@ -121,12 +121,6 @@ public void BeforeApplicationWindowPopup(BeforePopupEventArgs eventArgs)
...
-                {
-                    options.InitialState = "fullscreen";
-                }
-
-                options.Resizable = eventArgs.PopupFeatures.Resizable;
```
Comment (lneir) [inline minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindowManager.cs:129]: we should change this line to:
options.Resizable = true;

**Comment by lneir** [comment]
lgtm

**Pull request merged**